lib/dal/tests/integration_test/rebaser.rs

```diff
@@ -294,24 +291,30 @@ async fn func_node_with_arguments_conflict(ctx: &mut DalContext) {
         .expect("able to list args");
     assert_eq!(1, args.len());
 
-    let result = ChangeSetTestHelpers::apply_change_set_to_base(ctx).await;
-    assert!(matches!(
-        result,
-        Err(ChangeSetTestHelpersError::ChangeSetApply(_))
-    ));
-
-    if let Err(ChangeSetTestHelpersError::Transactions(TransactionsError::ConflictsOccurred(
-        conflicts,
-    ))) = result
-    {
-        assert_eq!(1, conflicts.conflicts_found.len());
-        let conflict = conflicts
-            .conflicts_found
-            .first()
-            .expect("conflict should be there")
-            .to_owned();
-        assert!(matches!(conflict, Conflict::ExclusiveEdgeMismatch { .. }));
-    }
+    // NOTE(nick): at the time of writing, this test has the "#![ignore]" macro applied and we've
+    // removed the formal "ChangeSetTestHelpersError" enum in favor of the "eyre!" and "Result"
+    // type. As an aside, that change was made to help make error reporting more clear for runtime
+    // assertion failures and explicit "expect" and "panic" usages in tests. Anyway, how the
+    // conflicts are detected will likely be slightly different in the future, so commenting this
+    // out for now should suffice. Please don't hesitate to reach out with questions or rage.
+    // let result = ChangeSetTestHelpers::apply_change_set_to_base(ctx).await;
+    // assert!(matches!(
+    //     result,
+    //     Err(ChangeSetTestHelpersError::ChangeSetApply(_))
+    // ));
+    //
+    // if let Err(ChangeSetTestHelpersError::Transactions(TransactionsError::ConflictsOccurred(
+    //     conflicts,
+    // ))) = result
+    // {
+    //     assert_eq!(1, conflicts.conflicts_found.len());
+    //     let conflict = conflicts
+    //         .conflicts_found
+    //         .first()
+    //         .expect("conflict should be there")
+    //         .to_owned();
+    //     assert!(matches!(conflict, Conflict::ExclusiveEdgeMismatch { .. }));
+    // }
```

Comment:
The comment here explains it further, but since this test currently has the `#[ignore] macro` and it is unclear what its future shape will be, we just comment out the assertion at the end.